### PR TITLE
Bump katello versions to 3.10

### PR DESCRIPTION
**Pull request by ehelms**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -499,6 +501,7 @@ katello_packages:
       build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
       nightly_package_tito_releaser_args:
         - "--arg jenkins_job=katello-master-release"
+        - "--arg jenkins_job_id=lastBuild"
```
Comment (ekohl) [inline package_manifest.yaml]: Why lastBuild instead of the last successful build (which we default to)?

File: package_manifest.yaml [hunk 2]
```diff
@@ -471,6 +471,8 @@ katello_packages:
       build_package_tito_releaser_args: "{{ nightly_package_tito_releaser_args }}"
       nightly_package_tito_releaser_args:
         - "--arg jenkins_job=katello-installer-nightly-release"
+        - "--arg jenkins_job_id=lastBuild"
```
Comment (ekohl) [inline package_manifest.yaml]: Same here

**Comment by ehelms** [comment]
[test rpm]

File: package_manifest.yaml (hunk 1 above)
Comment (ehelms) [reply package_manifest.yaml]: Chicken and egg problem the way those jobs work right now. They both build source and the RPM. However, if we are in this current state, the lastSuccessfulBuild will never point to a usable source. Further, we aren't validating the source, so we just need to most recently generated source.

Comment (ekohl) [reply package_manifest.yaml]: But `katello-installer-nightly-release` triggers a build with an explicit job ID so it'd be a matter of restarting that job. I don't think there's a need to pollute our history with this change and then changing it back. Other reasons for failure might be intermittent network failures. We also had jobs that timed out after 2 hours so had no artifact (more relevant on the installer, but still).

**Comment by ehelms** [comment]
This won't pass any tests due to chicken and egg problem so I'm going to merge it.

**Pull request merged**